src/seedu/addressbook/AddressBook.java

```diff
@@ -1150,8 +1204,8 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
+        return fullString.replace(prefix, "");
```

Comment:
as mentioned in tutorial, the replace behaviour would also remove instances of the `prefix` argument that are found else where than the start of `fullString`